jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/JavaLayerConfigurationsHelperTest.java

```diff
@@ -56,43 +62,59 @@ private static void assertExtractionPathsUnordered(
 
   @Test
   public void testFromExplodedWar() throws URISyntaxException, IOException {
-    Path explodedWar = Paths.get(Resources.getResource("exploded-war").toURI());
-    Files.createDirectories(explodedWar.resolve("WEB-INF/classes/empty_dir"));
+    // Copy test files to a temporary directory that we can safely operate on
+    Path temporaryExplodedWar = temporaryFolder.newFolder("exploded-war").toPath();
+    Path resourceExplodedWar = Paths.get(Resources.getResource("exploded-war").toURI());
+    try (Stream<Path> stream = Files.walk(resourceExplodedWar)) {
```

Comment:
We have `FileOperations#copy` that does the recursive copy - I believe we can use that here. There're also some other tests that do this kind of copy directory setup and we might want to use the `FileOperations#copy` method too so we don't have duplicate recursive copy implementations in various places.